src/Message.php

```diff
@@ -6,14 +6,10 @@
 
 final class Message implements MessageInterface
 {
-    /**
-     * @var string
-     */
+    /** @var string */
```

Comment:
How come not typed property?

Reply:
Yes, would be possible, but I did not change then typed properties itself in this PR. Maybe in another PR?

Reply:
ok, fair enough.